D/DuckDB/build_tarballs.jl

```diff
@@ -3,11 +3,11 @@
 using BinaryBuilder, Pkg
 
 name = "DuckDB"
-version = v"0.10.2"
+version = v"0.10.3"
 
 # Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/duckdb/duckdb.git", "1601d94f94a7e0d2eb805a94803eb1e3afbbe4ed"),
+    GitSource("https://github.com/duckdb/duckdb.git", "70fd6a8a2450c1e2a7d0547d4c0666a649dc378e"),
```

Comment:
Ok, now that the PR has been merged I can guess we can do something like
```suggestion
    GitSource("https://github.com/duckdb/duckdb.git", "3f33d34debb665b37c5e4d479afd7f85a64a94f7"),
```
?

Reply:
It should point to a nightly build - e.g. `1323e95677e571cfacd56e3dd1fe6d3220a76512` - otherwise there will be no extensions available